src/Rules/Drupal/LoadIncludes.php

```diff
@@ -48,6 +48,9 @@ public function processNode(Node $node, Scope $scope): array
         try {
             // Try to invoke it similarly as the module handler itself.
             [$moduleName, $filename] = $this->parseLoadIncludeArgs($args[0], $args[1], $args[2] ?? null, $scope);
+            if ($moduleName === null && $filename == null) {
+                return [];
+            }
```

Comment:
$moduleName could not be found (because it's a variable), proceeding to `$module = $this->extensionMap->getModule($moduleName);` would throw a `TypeError` with message "mglaman\PHPStanDrupal\Drupal\ExtensionMap::getModule(): Argument #1 ($name) must be of type string, null given, called in D:\git\phpstan-drupal\src\Rules\Drupal\LoadIncludes.php on line 54".

That would be caught by the `catch` later on in this class, which results in the error "A file could not be loaded from Drupal\Core\Extension\ModuleHandlerInterface::loadInclude", which is creating a false positive.

Bailing here makes it all better.

Reply:
Should `parseLoadIncludeArgs` be changed to not just return `[]` but an array of the same size each time when something is `null`?

This makes sense to just bail out.

Reply:
So something like my latest commit?

Reply:
Sense = made :)